### PR TITLE
fix: remove artifacts

### PR DESCRIPTION
**Pull request by ratik**

### CODE REVIEW

**Review by quasisamurai — APPROVED** [review]
c:

File: .gitignore [hunk 1]
```diff
@@ -13,4 +13,4 @@ Cargo.lock
 
 .DS_Store
 
-artifacts/checksums_intermediate.txt
+artifacts/
```
Comment (sotnikov-s) [inline .gitignore:16]: seems like we don't need the artifacts folder anymore

Comment (foxpy) [reply .gitignore:16]: what. why? It is an output folder for the build process and we better gitignore it.

Comment (ratik) [reply .gitignore:16]: no we need it to exclude it from git tracking

Comment (sotnikov-s) [reply .gitignore:16]: ah, alright, you're right

**Review by sotnikov-s — APPROVED** [review]

**Review by foxpy — APPROVED** [review]

**Review by oopcode — APPROVED** [review]

**Pull request merged**